### DISCUSSION TITLE
Discussion@git

### DISCUSSION THREAD

**Discussion by Saarthakgupta26** [arnav108276/Shopping_Mall_management_system#2] [General]
Hi everyone, please let me know when are you all free for the in person discussion ,if some problem we can continue our chat on this platform .

**Comment by Saarthakgupta26** [comment]
also please check the above code and let me know the feedback and ideas for the same.

**Comment by Saarthakgupta26** [comment]
> Hi everyone, please let me know when are you all free for the in person discussion ,if some problem we can continue our chat on this platform .

I am free after 6

**Reply by adityatomar25** [reply]
I'm also free

**Comment by Saarthakgupta26** [comment]
Checked the code we still have some errors . 

I like the color scheme please everyone share your views as well.

**Reply by arnav108276** [reply]
The error is in the version compatibility of sql and python I guess as earlier that particular section was working well but I reinstalled sql since then problem has occured.

**Reply by adityatomar25** [reply]
I'm having some issue in running the code, I think the issue is with sql connector. Can you help?

**Reply by arnav108276** [reply]
Change name of database, password, user with your database details

**Reply by adityatomar25** [reply]
Still not working

**Comment by Saarthakgupta26** [comment]
Good work Arnav and Rangoli

@rangolipanwar 
@arnav108276 
@yuvrajdixit 
@adityatomar25  
@CodewithStuti 
@ansh142536 
@simran-cynthia  

we are going present our project on 5th DEC 2023 , time isn't decided yet.

**Comment by arnav108276** [comment]
![image](https://github.com/arnav108276/Shopping_Mall_management_system/assets/143831459/61727601-b3b9-43ef-8d5f-54322664c258)
![image](https://github.com/arnav108276/Shopping_Mall_management_system/assets/143831459/ecd1c7e6-c35b-4ccc-9357-bb85565a0b5d)
![image](https://github.com/arnav108276/Shopping_Mall_management_system/assets/143831459/c7334bf3-ca2a-4203-b20f-c89e9f7b1af7)
![image](https://github.com/arnav108276/Shopping_Mall_management_system/assets/143831459/65baf157-c23f-41e5-9877-8c3f49dda4e6)

Guys this is the final output of our project.

**Comment by Saarthakgupta26** [comment]
We are going with our presentation as planned , I request everyone to be present tomorrow. No excuses will be tolerated for the same.

**Reply by adityatomar25** [reply]
Okay

**Comment by arnav108276** [comment]
Presentation at 12 noon today venue will be told
